src/pages/ReactLeafletMap/LeafletMap.jsx

```diff
@@ -0,0 +1,30 @@
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+
+import "../ReactLeafletMap/leaflet.scss";
+import "leaflet/dist/leaflet.css";
+
+export default function LeafletMap({ cordinates ,name}) {
+  
+  if (cordinates) {
```

Comment:
1 - Coordinates always true
2 - if u need some condition, first of all make empty return `if(!coordinates) { return  } `and then create ur component